Wikipedia/Code/TalkPageCell.swift

```diff
@@ -229,6 +229,10 @@ extension TalkPageCell: Themeable {
         rootContainer.backgroundColor = theme.colors.paperBackground
         rootContainer.layer.borderColor = theme.colors.border.cgColor
 
+        if theme == Theme.light {
```

Comment:
I'm sure we're not consistent with this, but I think we should avoid changes at the UI level like this. If there is a new combination of colors needed for this border, we should spin up a new semantic color group name and use that instead.

Reply:
Good point, I created a new one for this use case